docs/contributing.md

```diff
@@ -26,6 +26,16 @@ If you want to contribute and don't know where to start, here is a step-by-step
 6. Run `yarn` - install dependencies
 7. Run `yarn dev` - URL will be logged
 
+
+## Run designer and sdk in parallel
```

Comment:
"Run" is probably not very clear in this context, you mean more "develop designer and SDK" or "watch changes"

Reply:
Changed